docs/guides/troubleshooting.md

```
@@ -0,0 +1,951 @@
+   ```
+
+## Login & Authentication
+
+### Can't Create Account
+
+**Symptoms**: Sign-up button doesn't work
+
+**Solutions**:
+
+1. **Check Email Format**
+   - Use valid email format: user@domain.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)